README.md

```diff
@@ -12,26 +12,37 @@ Flash messages are typically used to display errors on websites that are rendere
 * [Examples](examples/)
 
 ```rust
-use actix_web::{http, server, App, HttpRequest, HttpResponse, Responder};
+use actix_web::{
+    web, 
+    App, 
+    HttpResponse, 
+    HttpServer,
+    HttpRequest,
+    Responder,
+};
 use actix_web_flash::{FlashMessage, FlashResponse, FlashMiddleware};
 
 fn show_flash(flash: FlashMessage<String>) -> impl Responder {
     flash.into_inner()
 }
 
-fn set_flash(_req: &HttpRequest) -> FlashResponse<HttpResponse, String> {
+fn set_flash(_req: HttpRequest) -> FlashResponse<HttpResponse, String> {
     FlashResponse::with_redirect("This is the message".to_owned(), "/show_flash")
 }
 
 fn main() {
-    server::new(|| {
+
+    HttpServer::new(move || {
         App::new()
-            .middleware(FlashMiddleware::default())
-            .route("/show_flash", http::Method::GET, show_flash)
-            .resource("/set_flash", |r| r.f(set_flash))
-    }).bind("127.0.0.1:8080")
+            .wrap(FlashMiddleware::default())
+            .route("/show_flash", web::get().to(show_flash))
+            .route("/set_flash", web::get().to(set_flash))
+
+    })
+        .bind("127.0.0.1:8080")
```

Comment:
This `bind` call should go on the previous line. Two other examples also have this, maybe just run rustfmt on the examples. :)

Reply:
I made this change to the README, but that is not what `cargo fmt` gave me (the `bind` on the previous line). If you have a `.rustfmt.toml` with your fmt preferences, you could check that in.

Reply:
Huh, not sure I am going to have a look at the formatting. Probably an old rust fmt version on my side.